bubbles/config.py

```diff
@@ -136,7 +138,8 @@ class AuthResponse:
     mods_array.append(None)
 
 # Import PluginManager from here
-PluginManager = PM(COMMAND_PREFIXES, INTERACTIVE_MODE)
+# PluginManager = PM(COMMAND_PREFIXES, INTERACTIVE_MODE)
+# PluginManager = None
```

Comment:
Why is this commented out? Can we remove this?

Reply:
whoops yeah